include/bluetooth/gatt.h

```diff
@@ -496,6 +496,13 @@ struct bt_gatt_ccc_cfg {
 	bt_addr_le_t		peer;
 	/** Config peer value. */
 	u16_t			value;
+	/** Config data		*/
+	union {
+		void		*data;
+		u32_t		data_u32;
+		u16_t		data_u16[2];
+		u8_t		data_u8[4];
+	};
```

Comment:
Since you're anyway prefixing these, why not give the union a name. That'd solve the CI errors you got as well (I think).

E.g:
`
union {
        void *ptr;
        u32_t u32;
        u16_t u16[2];
        u8_t  u8[4];
} data;`